README.md

```diff
@@ -11,3 +13,21 @@ For information on using the package,
 [see the stable documentation](https://gsa.sciml.ai/stable/). Use the
 [in-development documentation](https://gsa.sciml.ai/dev/) for the version of
 the documentation, which contains the unreleased features.
+[![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
+
+## Installation
+
+The GlobalSensitivity.jl package can be installed with julia's package manager as shown below:
+
+```julia
+]add GlobalSensitivity
+using GlobalSensitivity
+```
+
+## General Interface
+
+The general interface for performing global sensitivity analysis using this package is:
+
+```julia
+res = gsa(f, method, param_range; samples, batch=false)
```

Comment:
Put one example after this. We always do one code example with a nice plot in the README for any user facing package.